sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentAnalysisAudience.java

```diff
@@ -11,6 +11,13 @@
 /** Defines values for DocumentAnalysisAudience. */
 @Immutable
 public final class DocumentAnalysisAudience extends ExpandableStringEnum<DocumentAnalysisAudience> {
+
+    /**
+     * Constructs a DocumentAnalysisAudience object.
+     */
+    public DocumentAnalysisAudience() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.